awscli/customizations/wizard/wizards/iam/_new-role-wip.yml

```diff
@@ -107,5 +109,44 @@ plan:
         condition:
           variable: wants_config_profile
           equals: yes
-#execute:
+  preview:
+    shortname: Preview
+    description: Preview results
+    values:
+      preview_none:
+        type: template
+        value: ''
+      preview_cli_command:
+        type: template
+        value: |
+          aws iam create-role --role-name {role_name} --description {role_description} --assume-role-policy-document
+
+          aws iam attach-role-policy --role-name {role_name} --policy-arn {policy_arn}
```

Comment:
So one thing we may want to do is add backslashes to break up the command into multiple lines. Right now if your terminal is not wide enough it will wrap weirdly and breaking it up into multiple lines will make it easier to read as well. We can do this in a follow up PR as well.